app/src/main/java/org/simple/clinic/teleconsultlog/prescription/medicines/TeleconsultMedicinesView.kt

```diff
@@ -155,12 +155,15 @@ class TeleconsultMedicinesView(
   }
 
   override fun renderMedicines(
-      medicines: List<PrescribedDrug>,
-      medicineFrequencyToLabelMap: Map<MedicineFrequency?, DrugFrequencyLabel>
+      medicines: List<PrescribedDrug>
   ) {
     emptyMedicinesTextView.visibility = GONE
     medicinesRecyclerView.visibility = VISIBLE
 
+    val medicineFrequencyToLabelMap = drugFrequencyToLabelMap
```

Comment:
Hmm, this does not need to be computed every time this method is called, right? We can make it a lazy property of the class instead of doing this. What do you think?